### ISSUE TITLE
alma tests should be using get_pkg_data_filename

### ISSUE THREAD

**Issue by pllim** [astropy/astroquery#2062]
https://github.com/astropy/astroquery/blob/f7940f90a95649223ff466ad7374a88e3e755f2f/astroquery/alma/tests/test_alma.py#L18-L22

should be replaced by

```python
from astropy.utils.data import get_pkg_data_filename

your_test_data_file = get_pkg_data_filename('data/somefile.xml', package='astroquery.alma.tests')
```

**and**

the data files should be installed as package data for `'astroquery.alma.tests'` here:

https://github.com/astropy/astroquery/blob/f7940f90a95649223ff466ad7374a88e3e755f2f/astroquery/alma/setup_package.py#L10

Not sure how your CI is passing, but I cannot get `test_alma.py` to pass when running `python setup.py test` locally, as it cannot find a test data file (`alma/tests/data/alma-datalink.xml`).

**Comment by keflavich** [comment]
@andamian , can you take this?

**Comment by bsipocz** [comment]
This is fixed in #2056 already
